### PR TITLE
Background: fixed

### PR DESCRIPTION
**Pull request by mariaalouisaa**
Few things done here:
- Removed old button
- Canceled out test files (to get rid of errors in console)
- Made background fixed
- Made button fixed (this actually makes the lists appear next to it which I think looks cool!) 
- Gave the main element a margin top to eliminate positioning issue that arose after button was given fixed position.

Things to note:
- The margin top for the main will need to be given some media queries for positioning - but this can only really be done after the list is finished.
- Likewise the button could do with being smaller on mobile devices but will be easier to add media queries at the end :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,28 +17,26 @@ <h1>Binge Pad</h1>
     </header>
     <!-- background -->
     <div class="no-signal">
-    
-   <!-- button now within container of background styling -->
```
Comment (PJSalter) [inline index.html:21]: Thank You for removing the old button, this now fixes the error 🪂 🥳

File: index.html [hunk 2]
```diff
@@ -17,28 +17,26 @@ <h1>Binge Pad</h1>
...
-    <button id="new-list" class="tv">FUNKY TV IMAGE OR GIF!</button>
+    </div>
+  
+   <!-- button now outside container of background styling -->
+   <button aria-label="Add a new list" id="new-list" class="tv">
```
Comment (PJSalter) [inline index.html:23]: 🎊 🏁

File: index.html [hunk 3]
```diff
@@ -17,28 +17,26 @@ <h1>Binge Pad</h1>
...
-      </button>
 
-    </div>
 
 
     <!-- static class attribute 
     <div class="tv-static"></div>
     -->
-    <script src="test-helpers.js"></script>
+   <!-- <script src="test-helpers.js"></script> -->
```
Comment (PJSalter) [inline index.html:37]: Helpful that it's there for use later when we are all ready to create the tests 😸 🐰

File: style.css [hunk 4]
```diff
@@ -44,7 +44,8 @@ body {
 .no-signal {
   width: 100%; 
   height: 100%; 
-  position: relative;
+  position: fixed; /* changed to fixed */
+  z-index: -1;
```
Comment (PJSalter) [inline style.css:48]: Really found this interesting when you tested the z-index which kept the background continuing down with the list ✨ was super kool

File: style.css [hunk 5]
```diff
@@ -44,7 +44,8 @@ body {
 .no-signal {
   width: 100%; 
   height: 100%; 
-  position: relative;
+  position: fixed; /* changed to fixed */
```
Comment (PJSalter) [inline style.css:47]: Great method to fix the background issue 💯 🦜 🥳

File: style.css [hunk 6]
```diff
@@ -81,8 +85,8 @@ h1 {
 /* Styling for TV button */
 
 .tv {
-  margin: 0 2em;
-  position: relative;
+  margin: 3.5em 2em auto 2em;
+  position: fixed;
```
Comment (PJSalter) [inline style.css:89]: This is great 🙌

**Review by PJSalter — APPROVED** [review]

File: style.css (hunk 5 above)
Comment (AsmahanM) [reply style.css:47]: Amazing, thank you for fixing this

**Review by AsmahanM — APPROVED** [review]

**Pull request merged**